tasks/graylog_install.yml

```diff
@@ -1,19 +1,45 @@
 ---
-- name: Install Graylog (Ubuntu/Debian)
-  apt:
-    name: "{{ graylog_package_name }}"
-    state: present
-    update_cache: yes
+- block:
+  - name: DEB | Install Graylog (Ubuntu/Debian)
+    apt:
+      name: "{{ graylog_package_name }}"
+      state: present
+      update_cache: yes
   when: ansible_os_family == 'Debian'
 
-- name: Install Graylog (CentOS/RHEL)
-  yum:
-    name: "{{ graylog_package_name }}"
-    state: present
-  when: ansible_os_family == 'RedHat'
+- block:
+  - name: RH | Install Graylog (CentOS/RHEL)
+    yum:
+      name: "{{ graylog_package_name }}"
+      state: present
+
+  - name: RH | Install Graylog dependencies (CentOS/RHEL)
+    yum:
+      name: "{{ item }}"
+      state: present
+    with_items: "{[ graylog_rpm_dependencies ]}"
+  
+  - name: RH | Enable Graylog firewalld
```

Comment:
`name: RH | Enable Graylog firewalld` > `name: RH | Enable host firewalld`
to prevent miscommunication